src/store/participantsStore.js

```diff
@@ -860,6 +860,8 @@ const actions = {
 		await removeCurrentUserFromConversation(token)
 		// If successful, deletes the conversation from the store
 		await context.dispatch('deleteConversation', token)
+		context.dispatch('purgeParticipantsStore', token)
+		context.dispatch('cacheConversations')
```

Comment:
This action `removeCurrentUserFromConversation` is being called only when you leave the covnersation (either from settings or LeftSidebar)

Action `'deleteConversation'`, in opposite, is called, when we:
- leaving conversation
- deleting conversation
- handling error with joining

So, my opinion, should belong there, in conversationsStore